src/components/Button/index.tsx

```diff
@@ -7,24 +7,53 @@ import styles from './styles.module.css';
 export enum ButtonKind {
   /** A primary button, generally displayed in green. */
   PRIMARY = 'primary',
-  /** An inverted button style, generally displayed in white on a dark background. */
-  INVERTED = 'inverted',
   /** A button that only has a border. */
   OUTLINE = 'outline',
   /** A button which cannot be clicked, generally displayed in grey. */
   DISABLED = 'disabled',
+  /** A button which has no border, fits as a tertirary button. */
+  BORDERLESS = 'borderless',
+  /** A button that dispalys well on image distracting backgrounds */
+  SCREEN = 'screen',
 }
 
 export enum ButtonSize {
-  MEDIUM = 'medium',
-  SMALL = 'small',
+  SM = 'small',
+  MD = 'medium',
+  LG = 'large',
+}
+
+export enum ExtendedPadding {
+  XS = 'padding-xs',
+  SM = 'padding-sm',
+  MD = 'padding-md',
+  LG = 'padding-lg',
+  XL = 'padding-xl',
+}
+
+export enum AlternativeColors {
+  GREEN = 'green',
+  RED = 'red',
+  BLUE = 'blue',
+  YELLOW = 'yellow',
+  GREY = 'safe-grey',
+  PURPLE = 'safe-purple',
+}
+
+export enum HoverBehaviour {
+  LIGHTEN = 'bg-lighten',
+  DARKEN = 'bg-darken',
```

Comment:
I think this is no longer used?

Reply:
mm thanks! Latest commit has removed this bit !